### ISSUE TITLE
(Markup 4.0.092 - Word 2013) Programa mantém separadores que não foram tagueados e validação apresenta erro

### ISSUE THREAD

**Issue by leticiaquino** [scieloorg/PC-Programs#2388]
O programa Markup mantém no XML os separadores que não foram identificados no Markup e ao validar o arquivo, o editor de XML acusa erro.

No Markup:
![pontuacao-app](https://cloud.githubusercontent.com/assets/4397365/19475319/3c227ab0-9512-11e6-9554-ed64baed80ec.jpg)

No XML:
![pontuacao-em-app](https://cloud.githubusercontent.com/assets/4397365/19475335/50aa985a-9512-11e6-9512-7241cc45d372.jpg)

Verificar a possibilidade de excluir a pontuação que "sobra" entre as tags.

**Comment by leticiaquino** [comment]
O mesmo ocorre para lista de definição, pontuação do segundo apêndice e também ponto final após o parágrafo:

**Markup:**
![pontuacao-fora-elementos](https://cloud.githubusercontent.com/assets/4397365/19475611/cbe7be2a-9513-11e6-8246-fbf9dd0c3ec9.jpg)


**XML:**
![glossary-point](https://cloud.githubusercontent.com/assets/4397365/19475643/f89983f4-9513-11e6-87e7-da228c0b4610.jpg)

**Comment by robertatakenaka** [comment]
@leticiaquino : ok, mas acho que no caso, estes separadores fazem parte do título ou label. É um problema análogo ao que tínhamos com product.

**Comment by leticiaquino** [comment]
@robertatakenaka na verdade isso começou a ocorrer agora.
Antes da atualização do Markup o programa gerava o XML corretamente, não considerando a pontuação fora da tag.

**Comment by robertatakenaka** [comment]
@leticiaquino : o que quero dizer que é melhor incluir o separador nos devidos elementos. Senão vão "desaparecer" do texto final.

**Comment by leticiaquino** [comment]
@robertatakenaka Acontece que esses elementos não permitem, de acordo com a JATS, pontuação.
Nesse caso não podemos manter.

**Comment by robertatakenaka** [comment]
@leticiaquino : o Markup/XPM pode excluir qualquer coisa que esteja fora de elementos, a consequencia é que não apresentará no XML, logo não apresentará nem em HTML, nem ePub, nem em lugar nenhum, por isso, digo que faz parte do elemento.

Não é verdade que de acordo com a JATS não possa ter pontuação. Veja a definição de label (JATS 1.0):

> ``<label>``Label (of an Equation, Figure, Reference, etc.)

> Number and/or prefix word placed at the beginning of display elements (for example, equation, statement, figure).
> Remarks
> 
> As an example, a label placed at the beginning of an equation (<disp-formula>) could be a number such as ``<label>Equation 3.</label>, <label>(3)</label>, or <label>3.</label>``. A label before a statement such as a Proof or Lemma, could be a word such as ``<label>Proof</label> or <label>Hypothesis</label>``.
> 
> Many objects can take explicit labels. For example, a ``<label>`` can contain the number or symbol used as a prefix character for a footnote that is placed at the point of the footnote reference. Similarly, the number and any prefix word associated with the title or the caption of a <fig> can be marked as a <label>, for example, “Figure 3.”, “Exhibit 2.”, or “3.”.
> 
> **In Citations: A ``<label>`` can be used to preserve the prefix number for a reference or citation “25.” or “[Lapeyre 2010]”. This can be useful when reconstructing untagged cross-references**.
> 
> Accessibility: Sometimes a ``<label>`` needs to be pronounced in a way that is not reflected in its content or its tagging. The @alt attribute can be used to record the pronunciation for screen readers and other accessibility devices. For example, the element figure that follows might be pronounced as “Figure 3”.
> 
> ``<label alt="figure 3">Fig&amp;#x00A0;III.</label>``
>

**Comment by leticiaquino** [comment]
@robertatakenaka a tag ```<label>``` permite pontuação, estou falando do ```<glossary>```, ```<app>```  por exemplo.

Nós damos capacitações solicitando que os usuários do Markup não considerem a pontuação ao marcar determinado dado. Nesse sentido, o publico capacitado faz a marcação conforme o exemplo dado acima.

A questão é que o programa não está eliminando a pontuação que fica fora da tag e ao gerar o XML é apresentado erro.

**Comment by robertatakenaka** [comment]
@leticiaquino : sim, mas veja, se vc inclui o ponto que está fora de p, deixa de existir o ponto solto dentro de app, especialmente porque ponto final deve fazer parte do p, não?
O ponto antes de ``[/glossary]`` deveria estar dentro de defitem por exemplo. 

Se o programa simplemente omitir o usuário não vai perceber que não está considerando pontuação que deveria fazer parte do elemento.

**Comment by javani** [comment]
@robertatakenaka @leticiaquino no caso de glossary e def-list (term e def) não seria possível resolver com css? Os dois pontos não fazem parte nem do termo nem da definição. O único senão é ter diferença entre o separador que elegemos para a css e o apresentado no PDF. Acredito que dois pontos seja o mais comum.
O que vocês acham?
